superset/config.py

```diff
@@ -791,6 +793,20 @@ class D3TimeFormat(TypedDict, total=False):
     "CODEC": JsonKeyValueCodec(),
 }
 
+# Cache for thumbnail images. `CACHE_TYPE` defaults to `SupersetMetastoreCache`
+# that stores the values in the key-value table in the Superset metastore, as it's
+# required for Superset to operate correctly, but can be replaced by any
+# `Flask-Caching` backend.
+THUMBNAIL_CACHE_CONFIG = {
+    "CACHE_TYPE": "SupersetMetastoreCache",
+    "CACHE_DEFAULT_TIMEOUT": 300,
+    # Should the timeout be reset when retrieving a cached value?
+    "REFRESH_TIMEOUT_ON_RETRIEVAL": True,
+    # The following parameter only applies to `MetastoreCache`:
+    # How should entries be serialized/deserialized?
+    "CODEC": PickleKeyValueCodec(),
```

Comment:
I wonder if we should have a similar warning as this [one](https://github.com/apache/superset/blob/e79778ac6df9000080ec7ee5b24d59bc46e44423/superset/extensions/metastore_cache.py#L64).